common/v2/features/SendAssets/SendAssets.tsx

```diff
@@ -118,15 +117,14 @@ const getInitialState = (): ISendState => {
           time: Date.now(),
           chainId: 1
         },
-        network: undefined
+        network: undefined,
+        resolvedNSAddress: '', // Address returned when attempting to resolve an ENS/RNS address.
+        isResolvingNSName: false // Used to indicate recipient-address is ENS name that is currently attempting to be resolved.
       },
       isFetchingAccountValue: false, // Used to indicate looking up user's balance of currently-selected asset.
-      isResolvingNSName: false, // Used to indicate recipient-address is ENS name that is currently attempting to be resolved.
       isAddressLabelValid: false, // Used to indicate if recipient-address is found in the address book.
       isFetchingAssetPricing: false, // Used to indicate fetching CC rates for currently-selected asset.
       isEstimatingGasLimit: false, // Used to indicate that gas limit is being estimated using `eth_estimateGas` jsonrpc call.
```

Comment:
```
isFetchingAccountValue,
 isAddressLabelValid,
isFetchingAssetPricing
 isEstimatingGasLimit:
```

These states should be handled by their custom component, then only setting state when they have actually validated and fetched the correct value. So that The only state we worry about in FormData is `AccountValue`, `AddressLabel`, `AssetPricing`, and `GasLimit`